packages/backend/src/studio.ts

```diff
@@ -66,6 +67,16 @@ export class Studio {
 
   public async activate(): Promise<void> {
     console.log('starting AI Lab extension');
+    this.telemetry = env.createTelemetryLogger();
+
+    // Ensure version is above 1.8.0
+    if (!version || lt(version, '1.8.0')) {
```

Comment:
maybe read from package.json the value `engines.podman-desktop` instead of hardcoding it again <1.8.0 there

so at least check is consistent

Reply:
yes good idea, I updated it, and replace with `>=1.8.0` in the package json. I used then the satisfies method of the semver package.